conda_libmamba_solver/solver.py

```diff
@@ -331,7 +331,6 @@ def _log_info(self):
     def _setup_solver(self, index: LibMambaIndexHelper):
         self._solver_options = solver_options = [
             (api.SOLVER_FLAG_ALLOW_DOWNGRADE, 1),
-            (api.SOLVER_FLAG_INSTALL_ALSO_UPDATES, 1),
```

Comment:
Removing this flag so we can use `INSTALL` and a name-only spec as a way of saying "keep this package around". We have `UPDATE if installed else INSTALL` logic if all the INSTALL commands so this is ok.